resources/problem-vote.scss

```diff
@@ -0,0 +1,54 @@
+.vote-form {
```

Comment:
Why do we have both `.vote-form` and `#vote-form`?

Reply:
outdated comment
there's nothing left to do